R/crate.R

```diff
@@ -76,7 +77,13 @@ NULL
 #' # explicitly set its environment to the crate environment with the
 #' # set_env() function from rlang:
 #' crate(rlang::set_env(fn))
-crate <- function(.fn, ..., .parent_env = baseenv()) {
+crate <- function(
+  .fn,
+  ...,
+  .parent_env = baseenv(),
+  .error_arg = ".fn",
+  .error_call
```

Comment:
```suggestion
  .error_call = environment()
```

Though I'd really prefer `= current_env()` in this case (despite @DavisVaughan's suggested rule).